### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by Nemoron** [balenalabs/inkyshot#84]
Is this repository still actively being maintained?

**Comment by Botwrightt** [comment]
Hey, were you able to get it to work? I’m having issues right now. I put a issue thread as well

**Comment by Nemoron** [comment]
No. Since the pull request addressing this issue is over a year old and has not been merged, I think inkyshot is not being maintained.

**Comment by Botwrightt** [comment]
Ah okay thanks. I’m trying to get this or something similar to this as a gift if you of anything.
